Good-Bids/goodbids
# #18 Bid Products

# Summary

This PR creates a new Bid Product which will specifically be used to place bids for a new Auction, which happens automatically and is connected to the Auction via postmeta. Because this happens automatically, there's no reason to allow an Admin to set or modify this setting.

## Issues

* #18 
* #19 

## Testing Instructions

1. Create a new Auction
2. Fill in all the required fields
3. Publish the Auction
4. Visit the Products listing page
5. Verify there is a  new Bid Product with the Title and ID of the Auction.
6. The Bid Product should be in the "Bids" product category.

### Additional Reviewers

@nathan-schmidt-viget
